docs/docs/operate/profiling.md

```diff
@@ -0,0 +1,48 @@
+Mantis allows you to directly connect profiling tools to workers in order to do performance or memory profiling. This
+can be helpful for debugging OOMs. Below are the steps to start profiling your running Mantis job:
+
+ 1. Download a profiling tool. We recommend [Zulu Mission Control (ZMC)](https://www.azul.com/products/zulu-mission-control/).
+ 1. Untar ZMC
+    ```
+    tar -xzvf <zulu_mission_control_package>.tar.gz
+    ```
+ 1. Run zmc.
+    ```
+    note
+    On Mac the `Zulu Mission Control` app is actually a directory even though it has an icon.
+    The `zmc` executable is inside and needs to be called directly.
+    ```
+    ```
+    <your dir>/<zulu_mission_control_package>/Zulu\ Mission\ Control.app/Contents/MacOS/zmc
+    ```
+ 1. Find a worker you want to debug in your job and click on it to get its **ip address** and **debug port** In this
+    example these are `100.82.159.148` and `7156` respectively.
+
+    <img src="../images/profiling/workerDebugURL.png" alt="Worker Debug URL" width="600"/>
+
+ 1. Setup an SSH tunnel with local portforwarding for your worker and port. The tunnel command for the example worker
+    in the previous step would be:
+    ```
+    ssh -L 7156:100.82.159.148:7156 100.82.159.148
+    ```
+ 1. After setting up local port forwarding. Add the worker JVM to ZMC by clicking the plug and star icon in the top right corner.
+
+    <img src="../images/profiling/zmcAddVM.png" alt="ZMC add VM" width="600"/>
```

Comment:
Images can be done in markdown. Example: `![Job running](../images/sine_job_running.png)`

Reply:
I noticed that images get stretched weirdly with the markdown image reference in other mkdocs projects, so I've used `<img >` to control the width in the past. Will switch to using the markdown references, hopefully it'll come out okay.